manifests/4.7/sriov-network-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -298,8 +298,13 @@ spec:
                 name: sriov-network-operator
             spec:
               serviceAccountName: sriov-network-operator
-              nodeSelector:
-                node-role.kubernetes.io/master: ""
+              affinity:
```

Comment:
This is for openshift, we don't maintain this file upstream.

Reply:
+1 please drop this file change

Reply:
Done!